packages/OptionButton/src/OptionButton.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import {
+  TextIcon,
+  OptionScaleIcon,
+  MultipleChoiceIcon,
+  LikertIcon,
+} from './svgs';
+
+import './option-button.scss';
+
+export type ButtonType = 'Text' | 'OptionScale' | 'MultipleChoice' | 'Likert';
```

Comment:
For unions we use the camelCase
```suggestion
export type ButtonType = 'text' | 'optionScale' | 'multipleChoice' | 'likert';
```